hadongun/스택/탑.py

```diff
@@ -0,0 +1,39 @@
+n = int(input())
+top = list(map(int, input().split()))
```

Comment:
```suggestion
height = list(map(int, input().split()))
```

top 이라고 하니 stack의 top과 헷갈릴 여지가 있어서 height나 tower 정도로 네이밍을 수정하는게 괜찮아 보이네요...
어떻게 생각하시나요?